src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -0,0 +1,10 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import { IBasePickerSuggestionsProps, NormalPeoplePicker } from 'office-ui-fabric-react/lib/Pickers';
+
+/**
+ * PeoplePicker is simply NormalPeoplePicker
+ * [exported from Fabric](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker).
```

Comment:
Same thing here, I wouldn't mention the underlying implementation in a comment (it's obvious from the `import`), but rather explain what it does and when it should be used.